Maps part 2

Oporto opened the pull request:
Finished the requirements for the guided project: Maps part 2.
Things to improve upon for next version:
- Restraining the amount of markers added to the map (right now its all)
- Use geocode API to get location data on all universities for data set
- Style window that pops up when clicking on university

Oporto committed:
Added data set of university rankings

Added dataset from https://data.world/education/university-rankings-2017/workspace/project-summary with top-ranked American universities

Oporto committed:
Updated location on top 10 universities

Commit:
Added MapServlet.java to retrieve university locations for map

Commit:
Added map.html and fixed build error in Datastorage.java class

Commit:
Added clustering on map

Commit:
Added API key

Oporto committed:
Add files via upload

Commit:
Updated universities csv

Commit:
Base version of part 2 working

Commit:
Deliverable version - Guided projects - map part 2

mohamedally commented on src/main/java/com/google/codeu/servlets/UniversityDataServlet.java:
```diff
@@ -0,0 +1,69 @@
+            String line = scanner.nextLine();
+            String[] cells = line.split(",");
+            
+            System.out.println(line);
```
after testing this out maybe we should remove the printing??

Oporto replied:
I already tested it and forgot to remove it!

Oporto committed:
Removed local configuration settings for devserver

Removed local configuration settings for devserver that were accidentally pushed

Oporto committed:
Removed print statement

Removed print statement for testing in university data servlet

Oporto reviewed (commented):
Should be all set now
